settings/settings.go

```diff
@@ -34,12 +34,13 @@ type Settings struct {
 }
 
 // Load loads a new client from a settings file.
-func Load() (*Settings, error) {
-	filename := locateSettingsFile()
+func Load(cf string) (*Settings, error) {
+	filename := locateSettingsFile(cf)
 
 	if _, err := os.Stat(filename); err != nil {
 		if os.IsNotExist(err) {
-			return nil, errors.New("Not logged in. Use 'deis login' or 'deis register' to get started.")
+			return nil, fmt.Errorf(`Client configuration file not found at: %s
```

Comment:
I'm pretty sure this will break e2e. https://github.com/deis/workflow-e2e/blob/a31e7e659d90bfb253263b16f0b3d210fc643641/tests/auth_test.go#L33